server/api/core/email/email.js

```diff
@@ -27,27 +26,10 @@ export function send(options) {
  * @returns {Object} returns source
  */
 export function getTemplate(template) {
-  check(template, String);
```

Comment:
Searching for custom templates in the database before using the filesystem version has been restored.  The entire email API is now completely backward compatible.